### DISCUSSION TITLE
文件未打包

### DISCUSSION THREAD

**Discussion by FalconWu2017** [ofajs/discussion#21] [General]
新版本的ofa.min.js居然大了一倍，应该是没有打包吧。   
![image](https://github.com/user-attachments/assets/1fffe0c2-0969-4a40-8555-2de0aaa9e59f)

**Comment by kirakiray** [comment]
更新到最新版本试试：

https://cdn.jsdelivr.net/gh/kirakiray/ofa.js@4.5.23/dist/ofa.min.js

<img width="857" alt="image" src="https://github.com/user-attachments/assets/411eb42c-3c52-4074-b3a6-98d0749063bd">

**Reply by FalconWu2017** [reply]
嗯。新版的打包了。
